src/editors/content/learning/contiguoustext/utils.tsx

```diff
@@ -83,23 +83,24 @@ function findInputRef(editor: Editor, input: string): Maybe<Inline | Text | Bloc
 
 // Flexible find a node by a supplied predicate
 function findNodeByPredicate(editor: Editor,
```

Comment:
This function didn't work for nested nodes (it wouldn't find text nodes inside blocks), so I changed it to a dfs